internal/storage/bucket_handle_test.go

```diff
@@ -25,6 +25,10 @@ import (
 
 const missingObjectName string = "test/foo"
 
+// FakeGCSServer is not handling generation and metageneration checks for Delete flow.
+// Hence, we are not writing the test for this test case.
```

Comment:
we are not writing tests for these flows.

Reply:
fixed